src/products/Cart.java

```diff
@@ -0,0 +1,79 @@
+package products;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Cart {
+
+    List<Product> productsOnCart;
+
+
+    public Cart() {
+        this.productsOnCart = new ArrayList<Product>();
+
+    }
+
+
+    public boolean buy(Product p) {
```

Comment:
in questo metodo non devi passare un prodotto ogni volta che vuoi comprare dal carrello